outbound/hmail.js

```diff
@@ -270,6 +270,11 @@ class HMailItem extends events.EventEmitter {
             // duplicate each MX for each ip address family
             this.mxlist = [];
             for (const mx in mxlist) {
+                // Handle UNIX sockets for LMTP
+                if (mxlist[mx].path) {
+                    this.mxlist.push(mxlist[mx]);
+                    continue;
```

Comment:
This breaks with the style of the other part of the for loop. Just use an else if for the next line instead.

Reply:
Ok - done.